AlembicImporter/Packages/com.unity.formats.alembic/Runtime/Scripts/Misc/Bool.cs

```diff
@@ -1,7 +1,7 @@
 using System;
 using UnityEngine;
 
-namespace UTJ.Alembic
+namespace UnityEngine.Formats.Alembic.Sdk
 {
```

Comment:
The Sdk namespace depends on the Bool class, so I added the Bool class to the Sdk namespace